assets/app/views/browse/pod.html

```diff
@@ -86,6 +86,22 @@
               </tab>
               -->
 
+              <tab heading="Terminal" active="selectedTab.terminal"
+                select="renderOptions.preventContainerSockets = false"
```

Comment:
oh... that's a little hard to follow. could we do this instead:
`select="terminalTabSelected = true"`
...
`prevent="!terminalTabSelected && !selectedTab.terminal"`